### PR TITLE
Remove experimental AMD extensions.

### PR DESCRIPTION
**Pull request by msiglreith**
The extensions haven't being promoted to actual extensions within a long time frame and don't provide any value in its current state. I added them some long time ago but not using them anymore since I switched my IHV. As I guess noone else has use for them I would propose to just drop them.

### CODE REVIEW

**Comment by Ralith** [comment]
Seems reasonable to me, these look pretty niche and they cost maintenance effort.

File: Changelog.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Update Vulkan-Headers to 1.3.210 (#605)
 
+### Removed
+
+- Removed experimental AMD extensions.
```
Comment (MarijnS95) [inline Changelog.md:15]:
```suggestion
- Removed experimental AMD extensions (#607)
```

**Review by MarijnS95 — APPROVED** [review]
I can definitely sympathize with that. This extension has been a "pain" at times to update due to having code that mimicks what is autogenerated.

If anyone wants this extension back there should be some effort to write out the types and commands in "`vk.xml` format" and make that mergeable in the generator.

**Comment by MarijnS95** [comment]
Thanks!

**Pull request merged**